cmd/utils/flags.go

```diff
@@ -1563,11 +1564,16 @@ func SetEthConfig(ctx *cli.Context, stack *node.Node, cfg *eth.Config) {
 		cfg.EVMInterpreter = ctx.GlobalString(EVMInterpreterFlag.Name)
 	}
 	if ctx.GlobalIsSet(RPCGlobalGasCap.Name) {
-		cfg.RPCGasCap = new(big.Int).SetUint64(ctx.GlobalUint64(RPCGlobalGasCap.Name))
+		cfg.RPCGasCap = ctx.GlobalUint64(RPCGlobalGasCap.Name)
 	}
 	if ctx.GlobalIsSet(RPCGlobalTxFeeCap.Name) {
 		cfg.RPCTxFeeCap = ctx.GlobalFloat64(RPCGlobalTxFeeCap.Name)
 	}
+	if cap := cfg.RPCGasCap; cap != 0 {
```

Comment:
Perhaps we can simplify this to 

```go
if cfg.PRCGasCap != 0 {
    log something;
} else {
    log something;
}
```

And also would be nice to move this after the RPCGasCap setting(instead of after the txfee).